rpc/wrtc_call_queue_mongodb_test.go

```diff
@@ -14,14 +14,33 @@ import (
 	"go.viam.com/utils/testutils"
 )
 
+var activeAnswerersTrigger chan int
+
+func init() {
+	activeAnswerersTrigger = make(chan int, 10)
+}
+
+func activeAnswerersStub(hostnames []string) {
```

Comment:
@michaellee1019 this was my way to test that passing in a function would be called when running the call_queue. Not sure if a channel was the best way of going here but essentially I just wanted something that would be able to be pinged inside the operator goroutine.